packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -7,46 +7,63 @@ import 'package:flutter/foundation.dart';
 import 'deep_links_model.dart';
 import 'fake_data.dart';
 
-
 class DeepLinksController {
-
-  final _linkDatasNotifier = ValueNotifier<List<LinkData>>(allLinkDatas);
   ValueListenable<List<LinkData>> get linkDatasNotifier => _linkDatasNotifier;
+  ValueListenable<String> get searchContentNotifier => _searchContentNotifier;
+  ValueListenable<bool> get showSpitScreenNotifier => _showSpitScreenNotifier;
```

Comment:
style nit: order getters, private members, and setters per the guidelines here: https://github.com/flutter/devtools/blob/master/packages/README.md#order-of-getters-and-setters